Package.swift

```diff
@@ -0,0 +1,46 @@
+// swift-tools-version: 5.6
+import PackageDescription
+
+let package = Package(
+  name: "AirbnbSwiftFormatTool",
```

Comment:
```suggestion
  name: "AirbnbSwift",
```
I feel like we can take this approach to leave our options open if we want this package to do more.

Reply:
Good idea